src/lib/menu/menu-trigger.ts

```diff
@@ -197,12 +197,15 @@ export class MdMenuTrigger implements AfterViewInit, OnDestroy {
    */
   private _getPosition(): ConnectedPositionStrategy  {
     const positionX: HorizontalConnectionPos = this.menu.positionX === 'before' ? 'end' : 'start';
-    const positionY: VerticalConnectionPos = this.menu.positionY === 'above' ? 'bottom' : 'top';
-
+    const overlayY: VerticalConnectionPos = this.menu.positionY === 'above' ? 'bottom' : 'top';
+    let originY = overlayY;
+    if (this.menu.overlayTrigger) {
+      originY = overlayY === 'top' ? 'bottom' : 'top';
+    }
```

Comment:
If you want to overlap the trigger, shouldn't the originY and the overlayY be the same? e.g. the top of the trigger should connect to the top of the overlay or the bottom of the trigger should connect to the bottom of the overlay.  Let me know if I'm missing something.

```ts
let originY: VerticalConnectionPos;
if (this.menu.overlapTrigger) {
   originY = overlayY;
} else {
   originY = overlayY === 'top' ? 'bottom' : 'top';
}
```